CONTRIBUTING.md

```diff
@@ -145,15 +82,13 @@ If you want to run the documentation UI locally:
 
 ## Integration tests
 
-Currently, [starknet's test suite](https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/starknet/testing/starknet.py) has important differences with public networks. Like [not checking signature hints toward the end of the tx flow](https://github.com/OpenZeppelin/cairo-contracts/issues/386).
-
-That's why we strongly suggest testing new features against a testnet before submitting the PR, to make sure that everything works as expected in a real environment.
+Currently, starknet's test suite has important differences with public networks. We strongly suggest testing new features against a testnet before submitting the PR, to make sure that everything works as expected in a real environment.
```

Comment:
```suggestion
Currently, Starknet's test suite has important differences with public networks. We strongly suggest testing new features against a testnet before submitting the PR, to make sure that everything works as expected in a real environment.
```

Reply:
63? could not find anything

Reply:
My bad, I should have included the text in case the line changed. It's line 65 now in Contributing:
`*IMPORTANT* Please pay attention to the maintainer's feedback, since its` <- `its` should be `it's`